### DISCUSSION TITLE
NILLION APP PROGRAMMING FILES ON ANDROID PHONES

### DISCUSSION THREAD

**Discussion by Comfy242** [NillionNetwork/discussions#56] [General]
I realised phone users can install the nillion programming files  as it requires desktop.  nillion should as well bring in innovation on how phone users can run the application files without having to use a computer

**Comment by oceans404** [comment]
related to https://github.com/orgs/NillionNetwork/discussions/57

Binaries for the Nillion SDK, which are used by developers to create apps and tools on Nillion, are available for Apple (M1/M2), Apple (Intel chip), Linux (ARM chip), Linux (Intel/AMD chip) platforms: https://docs.nillion.com/limitations#platforms

End users can run blind apps on mobile if developers of the blind apps support mobile.
